e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -23,7 +23,14 @@
             "env": {
               "UpstreamProtocol": {
                 "value": "<UpstreamProtocol>"
+              },
+              "experimentalfeatures__enabled": {
```

Comment:
do you know what happen if edgeAgent is enabled for metrics?  edgeHub still will send out metrics, right?

Reply:
These flags are independent. You need both if you want both agent / hub to collect metrics.

Reply:
Although I assumed CollectMetrics was required for edgeHub, but it might not be (might only be required for MetricsV0)